lib/conjur/conjur_config.rb

```diff
@@ -23,6 +23,8 @@ class ConjurConfig < Anyway::Config
     attr_config(
       # Read TRUSTED_PROXIES before default to maintain backwards compatibility
       trusted_proxies: (ENV['TRUSTED_PROXIES'] || []),
+      # The maximum number of results for listing requests. The default value is 0 and means that there is no limit
```

Comment:
This comment needs to be wrapped: 

```suggestion
      # The maximum number of results for listing requests. The default value
      # is 0 and means that there is no limit.
```